templates/base.html

```diff
@@ -71,10 +63,16 @@
 			{% endblock %}
 		</main>
 		{% endblock %}
+
+		<!-- FOOTER -->
 		{% block footer %}
 			<footer>
-				<div class="info-button">
-					<a href="/info" title="View instance information">&#x24D8;</a>
+				<p id="version">v{{ env!("CARGO_PKG_VERSION") }}</p>
```

Comment:
Gave this some thought and I think we should wait on merging this until after I rework [`generate-instances-json.sh` in libreddit/libreddit-instances](https://github.com/libreddit/libreddit-instances/blob/3eb6c9124fa091f2fd5e550ed6fa89795d20b7a7/generate-instances-json.sh) to start using the instance info endpoint. If we merge this in now, [the version scraper will fail to pick up the version](https://github.com/libreddit/libreddit-instances/blob/3eb6c9124fa091f2fd5e550ed6fa89795d20b7a7/generate-instances-json.sh#L480) which will cause the script to reject the instance because the HTML tag with id `version` is now a `p` rather than a `span`.

Alternatively, we could change the script to accept either a `span` or a `p`, but really, we should probably be using that instance info endpoint to get the version, along with whatever the instance settings are (e.g. SFW-only mode).

Reply:
Or, better yet, have the script just look for `id=version` since we'll only have one HTML tag with that id.

Reply:
That's a good point. Yeah, let's wait on this PR until that script is reworked.

Reply:
@spikecodes: No need. I'll adjust the script to check for `id="version"` instead and that will take care of this problem.

Reply:
Script has been fixed. On that basis, I think this is good to go in.